src/chart/summaryTable/renderRow/makeBarChart.js

```diff
@@ -0,0 +1,152 @@
+import clone from '../../../histogram/util/clone';
+
+export function makeBarChart(this_, d) {
+    const chartContainer = d3.select(this_).node();
+    const chartSettings =
+        {x: {column: 'prop_n'
+            ,type: 'linear'
+            ,label: ''
+            ,format: '%'
+            ,domain: [0,null]}
+        ,y: {column: 'key'
+            ,type: 'ordinal'
+            ,label: ''}
+        ,marks:
+            [
+                {type: 'bar'
+                ,per: ['key']
+                ,summarizeX: 'mean'
+                ,tooltip: '[key]: [n] ([prop_n])'}
+            ]
+        ,gridlines: 'xy'
+        ,resizable: false
+        ,height: this_.height
+        ,margin: this_.margin
+        ,value_col: d.value_col
+        };
+
+    function onInit() {
+      //Add group labels.
+        if (this.config.group) {
+            const groupTitle = this.wrap
+                .append('p')
+                .text(`${this.config.group_col}: ${this.config.group} (n=${this.config.n})`);
+            this.wrap.node().parentNode
+                .insertBefore(groupTitle.node(), this.wrap.node())
+        }
+    }
+
+    function onResize() {
+      //Move y-axis labels to the right.
+        const ticks = this.wrap
+            .selectAll('g.y.axis g.tick');
+
+        ticks.select('text').remove();
+        ticks.append('title')
+            .text(d => d);
+        ticks
+            .append('text')
+            .attr(
+                {'text-anchor': 'start'
+                ,'alignment-baseline': 'middle'
+                ,'dx': '1em'
+                ,'x': this.plot_width})
+            .text(d => d.length < 30
+                ? d
+                : d.substring(0,30)+'...');
+
+      //Draw overall rates.
+        if (this.config.group) {
+            this.current_data
```

Comment:
Yeah, I think you described it right, but just to make sure we're on the same page: the position of the overall mark in grouped view for a given categorical value should match the position of the right side of the that categorical value's bar in the ungrouped view.
`
Ungrouped:
-----| F (60%)
---|   M (40%)
Grouped:
Site 1 (n=10)
-----|  F (50%)
---|-   M (50%)
Site 2 (n=10)
-----|- F (70%)
---|    M (30%)
`